resources/log4j.properties

```diff
@@ -20,5 +20,6 @@ log4j.appender.metabase=metabase.logger.Appender
 log4j.logger.metabase=INFO
 log4j.logger.metabase.db=DEBUG
 log4j.logger.metabase.driver=DEBUG
+log4j.logger.metabase.middleware=DEBUG
```

Comment:
i don't want them logged, not at debug level.  there is no reason to be that verbose for normal app execution.